sympy/core/containers.py

```diff
@@ -144,6 +145,37 @@ def index(self, value, start=None, stop=None):
         else:
             return self.args.index(value, start, stop)
 
+    @property
+    def kind(self):
+        """
+        Kind method is used to determine the type of Tuple objects.
+        Kind of Tuple will always be ``TupleKind`` with kinds of all arguments
+        as parameters
```

Comment:
Maybe change this to:

The kind of a Tuple instance.

The kind of a Tuple is parametrised by the number of elements and the kind of each element.